web3/utils/filters.py

```diff
@@ -180,6 +210,23 @@ def get(self, only_changes=True):
         ]
         return formatted_log_entries
 
+    def get(self, only_changes=True):
+        warnings.warn(DeprecationWarning(
+            "LogFilter.get has been deprecated and "
+            "will be removed from the LogFilter class in future releases. "
+            "Update your code to use the new methods: "
+            "LogFilter.get_new_entries and LogFilter.get_all_entries."
+        ))
+
+        self._ensure_not_running("get")
+
+        if only_changes:
+            log_entries = self.web3.eth.getFilterChanges(self.filter_id)
```

Comment:
Not necessary to change, but you *could* simply delegate to `self.get_new_entries(...)` here and `self.get_all_entries(...)`

Reply:
whoops, surprised I missed that 😁